doc/config.rst

```diff
@@ -878,10 +878,24 @@ that use Sphinx's HTMLWriter class.
 
 .. confval:: html_stylesheets
 
+   A list of CSS files.  The entry must be a *filename* string or a tuple
+   containing the *filename* string, the *alternate* (of boolean type) and
+   *title* (a string).  The *filename* must be relative to the
+   :confval:`html_static_path`, or a full URI with scheme like
+   ``http://example.org/style.css``.  It defaults to an empty list.
+
+   For more information about *alternate*, refer to the `MDN web docs`__.
+
+   __ https://mdn.io/Web/CSS/Alternative_style_sheets
+
    .. versionadded:: 1.8
 
 .. confval:: html_javascripts
 
+   A list of JavaScript *filename*.  The *filename* must be relative to the
```

Comment:
A list of JavaScript files.  The entry must be a *filename* string and must be relative...

?